modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -12,13 +12,15 @@ class ImagingUploader extends React.Component {
 
     this.state = {
       isLoaded: false,
-      filter: {}
+      filter: {},
+      hiddenHeaders: ['PatientName']
```

Comment:
is there a reason why hiddenHeaders is defined in both this.state.hiddenHeaders and this.state.data.hiddenHeaders? seems to me that we should use one or the other

Reply:
Yeah, good catch. When I commented out `this.state.hiddenHeaders` below, I accidentally added `data` to it out of personal confusion.